packages/core/src/Interfaces.ts

```diff
@@ -22,7 +22,7 @@ import {
 	IWorkflowSettings as IWorkflowSettingsWorkflow,
 } from 'n8n-workflow';
 
-import { OptionsWithUri, OptionsWithUrl } from 'request';
+// import { OptionsWithUri, OptionsWithUrl } from 'request';
 import requestPromise from 'request-promise-native';
```

Comment:
Shouldn't we also remove `request-promise-native`, as it's just a wrapper on top of `request` ?